src/composables/notifications.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  computed, ref, watch,
+} from '@vue/composition-api';
+import { INotification, INotificationSetting, NotificationStatus } from '../types';
+import {
+  NOTIFICATION_STATUS_CREATED,
+  NOTIFICATION_STATUS_READ,
+  NOTIFICATION_TYPE_WALLET,
+  executeAndSetInterval,
+  watchUntilTruthy,
+} from '../popup/utils';
+import { useGetter, useState } from './vuex';
+import { getAllNotifications, modifyNotifications, normalizeNotification } from '../popup/utils/notifications';
+
+const CHUNK_SIZE = 20;
+
+const superheroNotifications = ref<INotification[]>([]);
+const walletNotifications = ref<INotification[]>([]);
+const pollingInterval = ref<NodeJS.Timer | null>(null);
+
+export function useNotifications(root: any) {
+  const sdk = useState('sdk');
```

Comment:
Actually the true SDK state is in the plugin `sdkPlaugin`. So the line should look like this:
```
const sdk = useGetter('sdkPlugin/sdk');
```